.github/workflows/tests.yml

```diff
@@ -13,74 +12,83 @@ jobs:
           - 1.76
           - nightly
     steps:
-    - uses: actions/checkout@v4
-    - name: Build
-      run: cargo build --verbose
-    - name: Test
-      run: cargo test
-    - name: Test --no-default-features
-      run: cargo test --no-default-features --verbose
-    - name: Test --all-features
-      run: cargo test --all-features --verbose
+      - uses: actions/checkout@v4
+      - name: Install Rust ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          override: true
```

Comment:
Without that the matrix value was just ignored and it was always running on stable